geo_areas/models.py

```
@@ -37,6 +37,10 @@ class GeographicalArea(ValidityMixin):
     # This deals with subgroups of other groups
     parent = models.ForeignKey("self", on_delete=models.PROTECT, null=True, blank=True)
 
+    @property
+    def description(self):
+        return self.geographicalareadescription_set.last()
```

Hm. We might need to be a bit smarter here… is this:
- the most recent "database live" description?
- the most recent "database live" description that is `PUBLISHED` (so ignoring errors)?
- the "operationally live" description?

Do we know what the prototypes do for this?

---

Agreed, we're going to have to deal with this at some point. But I don't think we can until #25 is merged in - until then there is no concept of live, published etc. Only an ordered set, admittedly with a date attached.

I'd argue database live is what we want - considering that the users will be concerned with updating the latest data, not with what is operational.

Again can't do this till #25 is merged, so bit of a chicken and egg scenario. If #25 is merged I'll make the change. If this one goes first we can put it in a ticket.

---

And as I say this we merge in #25, I'll move it to database live - assuming that works for you.

---

I think we are going to have some more analysis to do here regardless. I think all of the above suggestions have problems! With "database live" there's a subtle line to walk about what states are included in that (maybe we could do with firming up that definition more generally). E.g. is `CDS_ERROR` included? If so, does that mean things that have errored (and under our model this is a permanent state) will for ever be hanging around in search results? 

Also, are we including stuff from the "active workbasket" here or is that to come later?

I think this is something in the medium term we need to revisit with user research, and ask what users actually expect to see. In the short term, we could decide to do what the current prototype does, but might find that makes no sense given that we've shuffled the way the states interact. So it sounds like we need to do some analysis work on this to work out if that still makes sense.

Let's not block this PR or more data model work on that though. Happy to go with something that seems sensible for now and I'll put a ticket in to revisit this.